composer.json

```diff
@@ -12,6 +12,9 @@
     "require": {
         "php": ">=5.3.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^4"
```

Comment:
It is require-dev, so isn't part of the deployment requirement. The use case for its presence is for a standalone package. I've not used symfony/phpunit-bridge, so can you show me what this should be please?